integration-tests/README.md

```diff
@@ -1,3 +1,10 @@
 # integration-tests Project
 
-WIP
\ No newline at end of file
+Module contains integration tests verifying end to end functionality in real cluster. The tests are executed using command `mvn clean verify -Pintegration'.
```

Comment:
May be better to use profile `cucumber` to keep the consistency with `shard-operator-integration-tests`.